cmd/fleetctl/gitops_test.go

```diff
@@ -2919,24 +2920,23 @@ software:
 }
 
 type memKeyValueStore struct {
-	m map[string]string
+	m sync.Map
```

Comment:
sync.Map is not a generic type, so we're untyped here and have a string cast on read.